onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -196,7 +203,7 @@ static bool IsTypeSupported(const NodeArg* node_arg) {
   }
 }
 
-static bool get_migraphx_type(ONNXTensorElementDataType type,
+static bool getMIGraphXType(ONNXTensorElementDataType type,
```

Comment:
Use capital G in GetMIG... and other places to be consistent with onnxruntime convention?